pkg/ccl/crosscluster/logical/dead_letter_queue_test.go

```diff
@@ -134,50 +134,56 @@ func TestDLQClient(t *testing.T) {
 
 	dbAName := "a"
 
-	tableNames := []fullyQualifiedTableName{
+	dstTableMd := []dstTableMetadata{
```

Comment:
i think we should make this data driven test slightly more readable: could you refactor the test cases such that a test case is:
```
type testCase{
  name string // describe what the test case tests
  dstTableMetadata
}
```

Then in the for loop below, you can call `t.Run()` for each test case

Reply:
I split the test into `TestDLQCreation` and `TestDLQLogging` as discussed offline